lib/register_client.rb

```diff
@@ -167,7 +168,13 @@ def update_data_from_rsf(rsf, user_entry_number, data_store)
     end
 
     def register_http_request(path)
-      RestClient.get(@register_url.merge(path).to_s)
+      headers = {}
+
+      if (@options[:api_key] != nil)
```

Comment:
are the parenthesis here redundant?

Reply:
Good point, although we use them interchangeably elsewhere throughout the code. It will most likely be worth having a PR dedicated to cleaning up the code, linting it to RuboCop properly etc.